wewatch-client/src/components/Join/Join.js

```diff
@@ -1,13 +1,37 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
 import { Link } from "react-router-dom";
 import queryString from 'query-string';
+import axios from 'axios'
 
 import './Join.css';
 import Layout from 'views/layout';
+import { Redirect } from "react-router-dom";
 
 const Join = ({location}) => {
   const [room, setRoom] = useState('');
   const { name } = queryString.parse(location.search);
+  const [validRoom, setValidRoom] = useState(false);
+
+  const checkRoom = () => {
+    axios.get('http://localhost:5000/getRoom').then(response => {
+      var rooms = response.data
```

Comment:
I think we should change these variables to `let` because it reduces the scope. Let me know what you think!

Reply:
+ 1 to that!

Reply:
sounds good!